src/main/resources/db/migration/V1_68__rename_nps_office_code_column_on_appointment_delivery.sql

```diff
@@ -0,0 +1 @@
+alter table appointment_delivery rename column nps_office_code to delius_office_location_code;
```

Comment:
Out of
- `nps_office_code`
- `delius_office_location_code`

the most semantically correct probably would be a third `probation_office_id` -- and using the reference data to make any dereferencing in the translation layer

Context: if we model our domain based on real-life, we're selecting an _office_, not a location code; our context doesn't require location IDs, that becomes relevant when we transition over the boundary to Delius' language

Reply:
ah so `probation_office_id` is the id on the first column in this csv? https://github.com/ministryofjustice/hmpps-reference-data/blob/main/probation-offices-v0.csv

I personally think it would be better with still using `delius_crs_location_id` for the following reasons:

1. I don't think it's safe to rely on an external data source id since we can't control if this changes. If we had the data out database this would be different since we can add constraints.
2. Similar to 1, I feel it's useful to have a record of the actual code we sent to delius. Yes, this is in reference-data but the connection is weaker and no protected from changes.
3. It means we don't have to implement any reference data integration on services side reducing the complexity for booking an appointment